packages/perseus/src/widgets/matrix.jsx

```diff
@@ -324,8 +325,14 @@ class Matrix extends React.Component<Props, State> {
                                             <NumberInput {...inputProps} />
                                         );
                                     } else {
+                                        const updatedProps = {...inputProps};
+                                        updatedProps.style = StyleSheet.create({
+                                            // eslint-disable-next-line react-native/no-unused-styles
+                                            input: inputProps.style,
+                                        }).input;
+
```

Comment:
**nit**: Given that you're already spreading `inputProps`, I think you could just do the style in that spread also... 
```suggestion
                                        const updatedProps = {
                                            ...inputProps, 
                                            style: StyleSheet.create({
                                                // eslint-disable-next-line react-native/no-unused-styles
                                                input: inputProps.style,
                                            }).input,
                                        };
```

Reply:
I'm going to do this in https://github.com/Khan/perseus/pull/332 to avoid a merge conflict